components/user-mgt/org.wso2.carbon.user.mgt.ui/src/main/resources/web/role/role-mgt.jsp

```diff
@@ -118,6 +133,20 @@
     }
 
 
+    String countFilter = request.getParameter(UserAdminUIConstants.ROLE_COUNT_FILTER);
+    if (countFilter == null || countFilter.trim().length() == 0) {
```

Comment:
Can use StringUtils.isBlank() in many more places. Please review entire PR for such places.